stats/cloud/api_test.go

```diff
@@ -0,0 +1,76 @@
+package cloud
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestCreateTestRun(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, `{"reference_id": "1"}`)
+	}))
+	defer server.Close()
+
+	client := NewClient("token", server.URL, "1.0")
+
+	tr := &TestRun{
+		Name: "test",
+	}
+	resp, err := client.CreateTestRun(tr)
+
+	if err != nil {
+		t.Fatal(err, nil)
```

Comment:
Please use the `github.com/stretchr/testify/assert` helpers; they generate helpful errors when assertions fail, and also help with more complex conditions, eg. map comparison (with errors printed as diffs).